src/main/java/seedu/toluist/controller/FindController.java

```diff
@@ -81,24 +84,30 @@ public CommandResult execute(String command) {
 
     private CommandResult formatDisplay(boolean isSearchByTag, boolean isSearchByName,
                                         String[] keywordList, int foundCount) {
-        String searchParameters;
-        if (isSearchByName && isSearchByTag) {
-            searchParameters = NAME_AND_TAG_MESSAGE;
-        } else if (isSearchByName) {
-            searchParameters = NAME_MESSAGE;
-        } else { //isSearchByTag
-            searchParameters = TAG_MESSAGE;
-        }
+        if (keywordList[0].equals(NULL_PARAMETER)) {
+            return new CommandResult(String.format(LIST_RESULT_MESSAGE_TEMPLATE, foundCount));
+        } else {
```

Comment:
should try to minimize your indentations whenever possible 
What I am saying here is that you can do this

```
if (if (keywordList[0].equals(NULL_PARAMETER)) {
        return new CommandResult(String.format(LIST_RESULT_MESSAGE_TEMPLATE, foundCount));
}
String keywords = .... // notice you dont need else here
```

This is one of the various ways to make your code look cleaner, and also make your guard statement stand out